.github/workflows/ci.yaml

```diff
@@ -0,0 +1,23 @@
+name: GH Actions CI
+on:      
+  pull_request:
+    branches:
+      - gh-pages
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
What does this do? The `actions/checkout@v2` portion, that is. Not used to the `@` syntax in these yet. (More familiar with Travis-CI tests).

Reply:
It checks out version 2 of the actions/checkout action -- I commented something to that effect. You can also pin to a specific commit or tag :)